src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariantContext.java

```diff
@@ -0,0 +1,280 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import htsjdk.samtools.*;
+import htsjdk.variant.variantcontext.StructuralVariantType;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFConstants;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceMultiSource;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.spark.sv.utils.GATKSVVCFConstants;
+import org.broadinstitute.hellbender.tools.spark.sv.utils.GATKSVVCFHeaderLines;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.haplotype.Haplotype;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+import org.broadinstitute.hellbender.utils.reference.ReferenceBases;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Supplier;
+
+/**
+ * Variant context with additional method to mind the structural variant specific information from structural
+ * variant records.
+ */
+public class StructuralVariantContext extends VariantContext {
```

Comment:
I'm gland that VC is not final otherwise I would need to use delegation... however as a matter of principle all classes should be final unless proven to be useful to be otherwise, so will change it to final.